frontend/app/components/Home.tsx

```diff
@@ -102,6 +99,11 @@ function Home (props: HomeProps) {
       <Typography component='footer' sx={{ textAlign: 'center' }}>
         Copyright © 2022 The Regents of the University of Michigan
       </Typography>
+      <Snackbar open={showRefreshAlert} onClose={handleRefreshAlertClose} autoHideDuration={10000}>
+        <Alert severity='success' elevation={2} onClose={handleRefreshAlertClose}>
+          Refresh the page to update the tool&apos;s appearance in the left-hand navigation.
```

Comment:
Is this the only alert we are needing able to handle? I'm mostly thinking if you'd use a Snackbar instead of the ErrorsDisplay components also? Looks good otherwise but I haven't tested it out.

Reply:
The other alerts are located within the card, or at the top of the page if the data fails to load. So I don't think there's the same usability problem, where the alert won't necessarily be visible.

That said, I take your point that it may be preferable to use alerts in a consistent manner. Do we think that errors should appear at the bottom of the page, and then maybe disappear?

Reply:
I'm gonna merge, but let me know what you think about this. If we did make the error messages use a snackbar, we could also use something like `notistack` as we did with CCM. We can open a separate issue if you think it's worth it.